### PR TITLE
[8.0][website_blog_excerpt_img] New layout for blog.

### PR DESCRIPTION
**Pull request by yajo**
Add an excerpt and an image taken from the post to the blog summary page:

To make this work, these open PR need to be merged:
- https://github.com/OCA/server-tools/pull/354
- https://github.com/OCA/server-tools/pull/355

### CODE REVIEW

File: website_blog_excerpt_img/README.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Excerpt + Image in Blog
+=======================
+
+This module extends the functionality of blogs to support having a more
+attractive layout that displays an excerpt of each post, a "Read more" button
+and an image. All automatic.
```
Comment (elicoidal) [inline website_blog_excerpt_img/README.rst]: Hi @Yajo Thanks for another interesting contribution!
Please remove or rephrase "All Automatic"

File: website_blog_excerpt_img/README.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+To install this module, you need to:
+
+* Install `OCA/server-tools <https://github.com/OCA/server-tools>`_.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to your blog.
```
Comment (elicoidal) [inline website_blog_excerpt_img/README.rst:25]: Wouldnot it deserve a little bit more?

File: website_blog_excerpt_img/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+"X-Generator: Poedit 1.8.6\n"
+
+#. module: website_blog_excerpt_img
+#: view:website:website_blog.blog_post_short
+msgid "Read more"
+msgstr "Leer más"
+
+#. module: website_blog_excerpt_img
+#: view:website:website_blog.blog_post_short
+msgid "col-xs-10"
```
Comment (elicoidal) [inline website_blog_excerpt_img/i18n/es.po:27]: Is it normal that these terms are translated?

File: website_blog_excerpt_img/README.rst (hunk 2 above)
Comment (yajo) [inline website_blog_excerpt_img/README.rst:25]: Sorry, but it's that simple.

File: website_blog_excerpt_img/i18n/es.po (hunk 3 above)
Comment (yajo) [inline website_blog_excerpt_img/i18n/es.po:27]: Yes, those are Bootstrap classes that get hit by the translator, but should not be modified.

**Comment by rafaelbn** [comment]
I set WIP until:
- https://github.com/OCA/server-tools/pull/354
- https://github.com/OCA/server-tools/pull/355

Are merged.

**Comment by rafaelbn** [comment]
Please @Yajo could you rebase? PRs are already merged. Let's check this PR

**Comment by rafaelbn** [comment]
Thanks! :+1:

**Comment by rafaelbn** [comment]
Thanks! :+1: @Tecnativa

**Comment by pedrobaeza** [comment]
:+1:

**Comment by paolovalier** [comment]
tested on runbot :+1:

**Pull request merged**